core/src/types/transaction/wrapper.rs

```diff
@@ -167,9 +167,6 @@ pub mod wrapper_tx {
         pub fee: Fee,
         /// Used to determine an implicit account of the fee payer
         pub pk: common::PublicKey,
-        /// The epoch in which the tx is to be submitted. This determines
-        /// which decryption key will be used
-        pub epoch: Epoch,
```

Comment:
If we remove it, we will have to put it back when the dkg comes online.

Reply:
Having spoken with @batconjurer the issue is now clear to me. I wasn't aware that with dkg we'll keep also old keys  so that we have the possibility to decrypt transactions coming from previous epochs. This means that we need to keep the `epoch` field since a transaction lifetime is not inherently limited to a single epoch. I'll revert this change